client/src/main/java/io/split/engine/sse/enums/CompressType.java

```diff
@@ -0,0 +1,17 @@
+package io.split.engine.sse.enums;
+
+public enum CompressType {
+    NOT_COMPRESSED(0),
+    GZIP(1),
+    ZLIB(2);
+
+    private long _value;
```

Comment:
Since properties in the other classes don't use the underscore prefix, maybe it could be omitted for consistency
```suggestion
    private long value;
```

Reply:
Updated